src/app/product-list-page/components/product-list-page/product-list-page.component.spec.ts

```diff
@@ -10,7 +11,8 @@ describe('ProductListPageComponent', () => {
   beforeEach(async(() => {
     TestBed.configureTestingModule({
       imports: [ SharedModule ],
-      declarations: [ ProductListPageComponent ]
+      declarations: [ ProductListPageComponent,
+      FiltersComponent ]
```

Comment:
It's better to indent it this way:
declarations: [
  ProductListPageComponent,
  FiltersComponent
]